indexer/db.go

```diff
@@ -1449,9 +1450,9 @@ func (p *PostgreSQLpgx) RetrievePathsAndBlockBounds(blockchain string, blockNumb
 
 	var paths []string
 
-	var minBlockNumber uint64
+	var minBlockNumber sql.NullInt64
```

Comment:
Why is it better?

Reply:
Can handle null values from database side propperly.